addons/stock_landed_costs/stock_landed_costs.py

```diff
@@ -286,7 +286,8 @@ def compute_landed_cost(self, cr, uid, ids, context=None):
 
                         if digits:
                             value = float_round(value, precision_digits=digits[1], rounding_method='UP')
-                            value = min(value, line.price_unit - value_split)
+                            fnc = min if line.price_unit > 0 else max
+                            value = fnc(value, line.price_unit - value_split)
```

Comment:
Ok for me, although I don't remember why this min/max is that necessary.  It is more like a safety feature.